MyFirstProject/LoginAndRegistrationApp/RegistrationViewClass.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  RegistrationViewClass.swift
+//  RegistrationApp
+//
+//  Created by Rishat on 17.09.2020.
+//  Copyright © 2020 Rishat Latypov. All rights reserved.
+//
+
+import UIKit
+
+protocol RegistrViewDelegate: AnyObject {
```

Comment:
Не нужно сокращать названия.

```suggestion
protocol RegistrationViewDelegate: AnyObject {
```